test/common/wpt.js

```diff
@@ -480,54 +572,58 @@ class WPTRunner {
       this.scriptsModifier?.(obj);
       scriptsToRun.push(obj);
 
-      const workerPath = path.join(__dirname, 'wpt/worker.js');
-      const worker = new Worker(workerPath, {
-        execArgv: this.flags,
-        workerData: {
-          testRelativePath: relativePath,
-          wptRunner: __filename,
-          wptPath: this.path,
-          initScript: this.fullInitScript,
-          harness: {
-            code: fs.readFileSync(harnessPath, 'utf8'),
-            filename: harnessPath,
+      for (const variant of meta.variant || ['']) {
```

Comment:
This is to align the "file names" with what the standard WPT runner executes. Some test files with thousands of individual are executed in smaller chunks with the variant `?1-1000`, `?1001-2000` attached to the URL of the file.

Reply:
Can you add a comment into the code for future reference?